content/assets/icon-packages.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Icon packages
+---
```

Comment:
On this page, I would move the description of each package right under the heading instead of after the image, and maybe make the formats available in a list.